packages/atomic/src/utils/accessibility-utils.tsx

```diff
@@ -56,7 +55,7 @@ export function FocusTarget() {
         }
         if (
           focusAfterSearch &&
-          getSearchUIDFromState(this.bindings) !== lastSearchId
+          this.bindings.store.getUniqueIDFromEngine(this.bindings.engine)
```

Comment:
here goes grandma Livia again... not sure if my comment disappeared or got resolved or if I never sent it, but better safe than sorry, so here it goes again:
are you missing `!== lastSearchId` here?
and: (for a future PR) would it make sense to save the engine in the store as well to make things simpler to access?

Reply:
Yep, fixed !